__tests__/server/utils/logging/development-formatters/index.spec.js

```diff
@@ -14,50 +14,41 @@
  * permissions and limitations under the License.
  */
 
-import yargs from 'yargs';
-
-jest.mock('yargs', () => ({ argv: {} }));
+jest.mock('../../../../../src/server/utils/logging/development-formatters/verbose', () => 'verbose export');
+jest.mock('../../../../../src/server/utils/logging/development-formatters/friendly', () => 'friendly export');
```

Comment:
`./src/server/utils/logging/development-formatters/index.js` conditionally exports `verbose` or `friendly` and I am just testing that branching logic. Mocking away the actual contents of the export as they are not needed for this test case.